tests/eas/load_tracking.py

```diff
@@ -1273,3 +1290,224 @@ def test_util_tg4_group(self):
+        return self._test_group_util('/tg1')
+
+    def test_util_tg2_group(self):
+        """Test utilization propagation to cgroup /tg2"""
+        return self._test_group_util('/tg2')
+
+class NestedCgroupsMigrationTest(_PELTTaskGroupsTest):
+    """
+    Test PELT utilization for task groups migration.
+    Initial group hierarchy:
+                            +-----+
+                            | "/" |
+                            +-----+
+                             /
+                            /
+                       +------+
+                       |"/tg1"|
+                       +------+
+                         /
+                        /
+                +----------+
+                |"/tg1/tg2"|
+                +----------+
+                   /    \
+                  /      \
+                t2_1    t2_2
+
+    Final group hierarchy:
+                            +-----+
+                            | "/" |
+                            +-----+
+                             /
+                            /
+                       +------+
+                       |"/tg1"|
+                       +------+
+                         /   \
+                        /     \
+                +----------+ t2_2
+                |"/tg1/tg2"|
+                +----------+
+                   /
+                  /
+                t2_1
+    """
+    target_cpu = 0
+    root_group = None
+    trace = None
+
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        # Run all workloads on a CPU with highest capacity
+        cls.target_cpu = [min(test_env.calibration(),
+                              key=test_env.calibration().get)]
+        period_ms = 16
+        duration_s = 4
+        # Create taskgroups
+        cpus = test_env.target.list_online_cpus()
+        cls.root_group = Taskgroup("/", cpus, 0, test_env)
+        tg1 = Taskgroup("/tg1", cpus, 0, test_env)
+        tg2 = Taskgroup("/tg1/tg2", cpus, 0, test_env)
+
+        # Create tasks
+        t2_1 = Task("t2_1", test_env, cls.target_cpu, period_ms=period_ms,
+                    duty_cycle_pct=10, duration_s=duration_s)
+        t2_2 = Task("t2_2", test_env, cls.target_cpu, period_ms=period_ms,
+                    duty_cycle_pct=20, duration_s=duration_s)
+
+        # Link nodes to the hierarchy tree
+        cls.root_group.add_children([tg1])
+        tg1.add_children([tg2])
+        tg2.add_children([t2_1, t2_2])
+
+        conf = {
+            'tag' : 'nested_cgp_migration',
+            'flags' : ['ftrace', 'freeze_userspace'],
+            'cpufreq' : {'governor' : 'performance'},
+        }
+
+        return {
+            'wloads': {},
+            'confs': [conf],
+        }
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(NestedCgroupsMigrationTest, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def _migrate_task(cls, test_env):
+        exclude = ''
+        cgroups = test_env.target.cgroups
```

Comment:
that's just because of my OCD: can we swap these two lines... it seems to read better to me :)
... sorry :(

Reply:
Kind of done: exclude = '' has been removed